stdlib/public/runtime/Errors.cpp

```diff
@@ -98,7 +101,7 @@ static bool getSymbolNameAddr(llvm::StringRef libraryName,
                                 sizeof(szUndName), dwFlags);
   });
 
-  if (dwResult == TRUE) {
+  if (dwResult != 0) {
```

Comment:
Personally, I'd prefer `if (dwResult) {`

Reply:
FWIW `if (dwResult)` would match the general LLVM/Swift style as well.

Reply:
Huh, really? It's not testing for truth though, it's testing for a non-zero value. But sure, I can change this.

Reply:
Changed it.